build.gradle

```diff
@@ -32,7 +32,7 @@ plugins {
     id 'com.gradle.build-scan' version '2.0.2'
     id 'base'
 }
-if (properties.get("org.elasticsearch.acceptScanTOS", "false") == "true") {
+if (project.properties.get("org.elasticsearch.acceptScanTOS", "false") == "true") {
```

Comment:
We should never be directly accessing `properties` in any case. This is not a Gradle API, it's a general Groovy mechanism. It just happens to work with Gradle properties via `propertyMissing()`.

Instead we should use the available Gradle APIs here. I'd change this to something like this:
```
if (Boolean.valueOf(project.findProperty('org.elasticsearch.acceptScanTOS') ?: "false"))
```